e2e_tests/tests/test_system.py

```diff
@@ -504,3 +504,10 @@ def test_perform_initial_validation() -> None:
         config, conf.fixtures_path("no_op"), 1, has_zeroth_step=True
     )
     exp.assert_performed_initial_validation(exp_id)
+
+
+@pytest.mark.e2e_cpu  # type: ignore
```

Comment:
What code gets tested here, beyond the tiral workload sequencer?  That is: why make this an e2e test when we could make it a unit test?

Reply:
Agree, just swapped in a unit test.